source/includes/connect/tls.c

```diff
@@ -0,0 +1,62 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  mongoc_init ();
+
+  {
+    // Set TLS using connection string
+    // start-connect-str
+    mongoc_client_t *client = mongoc_client_new ("mongodb+srv://<db_username>:<db_password>@<hostname/port>/?tls=true");
```

Comment:
```suggestion
    mongoc_client_t *client = mongoc_client_new ("mongodb+srv://<db_username>:<db_password>@<hostname>/?tls=true");
```

Quoting [Initial DNS Seedlist Discovery](https://github.com/mongodb/specifications/blob/c5dcae52a77d1ac90ca8c0cea7f371b73982062e/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md):
> It is an error to specify a port in a `mongodb+srv` connection string.